packages/angular/src/builders/delegate-build/schema.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface DelegateBuildExecutorSchema {
+  buildTarget: string;
+  outputPath: string;
+  tsConfig: string;
```

Comment:
I understand that `tsConfig` is required for the Incremental Build to work. Why do we need `outputPath`? should that not be defined in the `buildTarget`'s options?